pkg/chunk/cached_store.go

```diff
@@ -39,6 +40,23 @@ const SlowRequest = time.Second * time.Duration(10)
 
 var (
 	logger = utils.GetLogger("juicefs")
+
+	cacheHits = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "juicefs_blockcache_hits",
+		Help: "read from cached block",
+	})
+	cacheMiss = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "juicefs_blockcache_miss",
+		Help: "missed read from cached block",
+	})
+	cacheHitBytes = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "juicefs_blockcache_hit_bytes",
+		Help: "read bytes from cached block",
+	})
+	cacheMissBytes = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "juicefs_blockcache_miss_bytes",
+		Help: "missed bytes from cached block",
+	})
```

Comment:
👋 This is nitpicking but for the first two I think it should be `juicefs_blockcache_hits` & `juicefs_blockcache_misses` (or `juicefs_blockcache_hit_count` & `juicefs_blockcache_miss_count`)

Right now `hits` is plural but `miss` is singular 

for example: https://github.com/oliver006/redis_exporter/blob/d0597c841d2c9fa30ce8b6ded6251d1994822e27/exporter/exporter.go#L168-L169